internal/bminventory/inventory.go

```diff
@@ -148,7 +149,16 @@ const ignitionConfigFormat = `{
           "name": "root"
       },
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-	}{{if .RH_ROOT_CA}},
+	},
+	{
+		"overwrite": true,
+		"path": "/root/.docker/config.json",
```

Comment:
btw, why didnt we do it with auth?

Reply:
What do you mean?

Reply:
I guess he meant the assisted-service-client auth.
Net relevant for this PR